### PR TITLE
Nodejs/week3

### PR DESCRIPTION
**Pull request by BikramRawat**
nodejs/week3

### CODE REVIEW

**Comment by BikramRawat** [comment]
Only two files: concerts.js and index.js file are for review , other files are just added as an template, since i pushed entire folder, thanks :)

File: nodejs/week3/warmupExercise/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({extended: true}));
+
+app.get("/", (req, res) => res.send("Welcome to Nodejs week3 warmup homework"));
+
+// Multiplication
+// http://localhost:3000/calculator/multiply?firstParam=2&secondParam=4&thirdParam=5
+// response: Multiplication of given numbers = 40
+app.get('/calculator/multiply', (request, response) => {
```
Comment (NatsGt) [inline nodejs/week3/warmupExercise/index.js:12]: Hello, nice work you did here, just a small observation, on the homework description you are asked to be able to get this result `/calculator/multiply?firstParam=1&secondParam=2&secondParam=4` => 8, but you get an error, try to figure out how to solve it.

File: nodejs/week3/classExercise/src/backend/api/concerts.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
+// const { request, response } = require("express");
+const express = require("express");
```
Comment (NatsGt) [inline nodejs/week3/classExercise/src/backend/api/concerts.js:2]: The homework only asked for the calculator and the backend for the meal sharing app, so please remove all the concert related files.

File: nodejs/week3/warmupExercise/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        if(isNaN(queryParam[parameter])){
+            return response.status(406).send('Unacceptable Input Parameter, Plz give numbers 😊');
+        }
+        result = result + Number(queryParam[parameter]);
+    }
+    response.send(`The sum of given numbers = ${result}`);
+})
+
+// Getting data through the request body i.e request.body using POST (hint: use postman)
+app.post('/calculator/multiply', (request, response) => {
```
Comment (NatsGt) [inline nodejs/week3/warmupExercise/index.js:38]: I would expect to have a get method and a post method for each math operation.

Comment (BikramRawat) [reply nodejs/week3/warmupExercise/index.js:38]: Thanks @NatsGt  for the review and the feedbacks, i will update them :)

**Pull request merged**